youtube_dl/extractor/pbs.py

```diff
@@ -14,10 +14,10 @@
     orderedSet,
     strip_jsonp,
     strip_or_none,
+    try_get,
     unified_strdate,
     url_or_none,
-    US_RATINGS,
-)
+    US_RATINGS)
```

Comment:
I missed subs on PBS, so I'm really interested in getting this merged. I'm not in any way familiar with the detailed coding guidelines and can't speak for dstftw,  but I think this change of formatting of the list might be an issue (removing the comma and moving up the closing bracket ).